Templates/Ros2ProjectTemplate/Docker/README.md

```diff
@@ -0,0 +1,118 @@
+# Docker scripts for running the O3DE Ros2Project based on O3DE Ros2ProjectTemplate
```

Comment:
Please mention the `cleanup.bash` file in the readme and describe its purpose.

Reply:
Readme updated; script added to the docker build process.